src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -114,16 +138,33 @@ private void validateCeasedDate(OverseasEntitySubmissionDto overseasEntitySubmis
         }
     }
 
-    private boolean isTrustStillRequired(OverseasEntitySubmissionDto overseasEntitySubmissionDto, String trustId, boolean istrustInvolvedInOverseasEntity) {
+    private boolean noBeneficalOwenersArePresent(OverseasEntitySubmissionDto overseasEntitySubmissionDto) {
```

Comment:
Spelling. `noBeneficialOwnersArePresent()`.

Reply:
Corrected the spelling